src/ring_theory/adjoin/basic.lean

```diff
@@ -241,6 +241,12 @@ def adjoin_comm_semiring_of_comm {s : set A} (hcomm : ∀ (a ∈ s) (b ∈ s), a
   end,
   ..(adjoin R s).to_semiring }
 
+lemma adjoin_singleton_one : adjoin R ({1} : set A) = ⊥ :=
```

Comment:
Can you generalize this to units? Thanks!

Reply:
I don't really follow -- what statement do you have in mind?

Reply:
Something like `djoin R ({u} : set A) = ⊥` for `u : Aˣ`. It should be easy.

Reply:
But this is false: this is the algebra generated by `u`, which not the trivial algebra generated by the identity alone.

Reply:
Ahahah, I am sorry!!